generated/nidaqmx/_linux_installation_commands.py

```diff
@@ -0,0 +1,84 @@
+import click
+from typing import List, Dict, Callable
+
+def _get_version_ubuntu(dist_version: str) -> str:
+    return dist_version.replace(".", "")
+    
+def _get_version_opensuse(dist_version: str) -> str:
+    return dist_version.replace(".", "")
+
+def _get_version_rhel(dist_version: str) -> str:
+    return dist_version.split(".")[0]
+
+# Command templates
+APT_INSTALL_COMMANDS = [
+    ["sudo", "apt", "update"],
```

Comment:
don't use use sudo in the commands -> what if they're running as root and don't have sudo installed?

instead, just list the commands (`apt update`, `apt install ...`, and document on the command that it needs to be run with super privileges (i.e., as root, or with sudo)

Reply:
@mshafer-NI , I have removed the sudo now and we will update the document.